organize/models.py

```diff
@@ -104,9 +105,10 @@ def deploy(self):
 
         self.change_status_to(DEPLOYED)
 
-        # TODO: we should recognize here if we should create a new event,
-        # copy old one or copy old and change organizaers.
-        event = self.create_event()
+        if self.previous_event:
```

Comment:
Yeah, but at least we should create a simple solution here, and then address edgecases later. Now it's just plain wrong?